tests/integration/components/file-menu-test.js

```diff
@@ -0,0 +1,130 @@
+import Ember from "ember";
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent('file-menu', 'Integration | Component | file menu', {
+  integration: true,
+  beforeEach() {
+
+    this.addFileCalled = false;
```

Comment:
This is totally fine as is, but removing these would result in the same test (since `this.addFileCalled` would be undefined and not truthy the assertions would still pass).